src/WebsocketServer.js

```diff
@@ -203,9 +180,30 @@ module.exports = class WebsocketServer extends events.EventEmitter {
     }
 
     /* eslint-disable class-methods-use-this */
-    handleResendRequestV0(connection) {
-        connection.sendError('ResendRequestV0 is not supported anymore. Please update your Streamr client.')
-        return null
+    handleResendRequestV0(connection, request) {
+        if (request.resendOptions.resend_last != null) {
+            this.handleResendRequest(connection, request, () => this.storage.fetchLatest(
```

Comment:
Would it be more DRY to handle `V0` requests by mapping them to `V1` requests and calling the appropriate handler method for the current messages? Then we wouldn't need to replicate the `this.storage.xxxx` logic here.
```
if (request.resendOptions.resend_last != null) {
    const newReq = ... // Create latest version resend request via ResendLastRequest.create(..) or similar
    return this.handleResendLastRequest(connection,newReq)
}
```